chaos_kitten/brain/orchestrator.py

```
@@ -78,6 +79,45 @@ def parse_openapi(state: AgentState) -> Dict[str, Any]:
+            f"relevant endpoints[/green]"
+        )
+        
+        if nl_plan.get("focus"):
+            console.print(f"[yellow]Focus:[/yellow] {nl_plan['focus']}")
+        
+        return {
+            "endpoints": filtered_endpoints or state["endpoints"],  # Fallback to all if none match
+            "nl_plan": nl_plan
+        }
+    except Exception as exc:
+        logger.exception(f"Natural language planning failed: {exc}")
```

Ruff (0.15.1)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)